app/views/performance_dashboards/_pdf_download.haml

```diff
@@ -0,0 +1,48 @@
+%button.btn.btn-secondary.j-download-pdf(type='button') Download PDF
+
+#j-download-modal.modal.fade(role="dialog" aria-hidden="true")
+  .modal-dialog.modal-dialog-scrollable.modal-dialog-centered{role: :document}
+    .modal-content
+      .modal-header
+        %h4.modal-title Download PDF
+        %button.close(aria-label="Close" type="button" data-dismiss="modal")
+          %i.modal-icon-close
+      .modal-body.clearfix
+
+
+- content_for :page_js do
+  :javascript
+    'use strict';
+    var url = #{url_for(format: 'pdf').to_json.html_safe};
+    var processResponse = function(data) {
+      var blob = new Blob([data], {type: 'application/pdf'});
+      var link = document.createElement('a');
+      link.href = window.URL.createObjectURL(blob);
+      link.download = "download.pdf";
+      link.click();
+    }
+    var submitForm = function($form) {
+      var xhr = $.ajax({
+        type: 'GET',
+        url: url,
+        xhrFields: { responseType: 'blob' },
+        data: $form.serialize(),
+      });
+      var cancel = function() { xhr.abort() }
+      return {xhr: xhr.then(processResponse), cancel: cancel}
+    }
+    $(function() {
+      var $modal = $("#j-download-modal");
+      $(".j-download-pdf").click(function() {
+        $modal.find('.modal-body').html('<p class="lead">Processing...</p>');
+        $modal.modal('show');
+        var $form = $(this).closest('form');
+        var ajax = submitForm($form)
+        ajax.xhr
+          .then(function() { $modal.modal('hide') } )
+          .catch(function() {
+            $modal.find('.modal-body').html('<p class="lead text-danger">An Error Occured, please try again</p>');
+          });
+        $modal.on('hidden.bs.modal', ajax.cancel)
+      });
+    });
```

Comment:
This is probably coming in the next iteration, but I think this should get moved off into its own JS file over in assets.  Then we can either drop in a class that it looks for, or a single line or two here to initialize the functionality.